src/EventStore.Core/Services/Transport/Grpc/Enumerators.ReadAllBackwards.cs

```diff
@@ -76,7 +79,7 @@ public class ReadAllBackwards : IAsyncEnumerator<ResolvedEvent> {
 
 				_bus.Publish(new ClientMessage.ReadAllEventsBackward(
 					correlationId, correlationId, new CallbackEnvelope(OnMessage),
-					commitPosition, preparePosition, Math.Min(_maxCount, 32),
+					commitPosition, preparePosition, 32,
```

Comment:
IIRC I put that in there because I had confused max count with page size. To me it only makes sense if someone is trying to read the all stream with a page size of 1, which they can't do now